openhands_cli/agent_chat.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python3
```

Comment:
@openhands please redo this file; use a similar pattern to the Openhands repo for cli related code instead of wrapping the `CodeActAgent` in another agent class